package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "lb4-log-extension-example",
+  "version": "1.0.0",
+  "description": "An example extension project for LoopBack Next",
+  "main": "index.js",
+  "engines": {
+    "node": ">=8"
+  },
+  "scripts": {
+    "build": "npm run build:lib && npm run build:lib6",
+    "build:lib": "tsc --target es2017 --outDir dist",
+    "build:lib6": "tsc --target es2015 --outDir dist6",
```

Comment:
Should this be `build:lib` and `build:lib6`? 

Looks like loopback-next-extension-starter needs to be fixed as well: https://github.com/strongloop/loopback-next-extension-starter/blob/02dc502034166d23a0cbff8907430b4fd861adb5/package.json#L10-L12

Reply:
To be consistent, it should be `build:dist` and `build:dist6`.